CHANGELOG.md

```diff
@@ -54,12 +54,13 @@ Fix `STATUS_HEAP_CORRUPTION` crash when concurrently calling `create_sampler`. B
 
 - Fixed memory leak in vulkan backend. By @cwfitzgerald in [#8031](https://github.com/gfx-rs/wgpu/pull/8031).
 
-## v26.0.2 (2025-07-26)
+## v26.0.2 (2025-07-23)
```

Comment:
@cwfitzgerald: Out of curiosity…how did this happen? I don't think any harm came of it, since I haven't observed anybody being confused about it.